### ISSUE TITLE
Add Collection to the Postman API Network

### ISSUE THREAD

**Issue by kinlane** [forcedotcom/postman-salesforce-apis#20]
We would love to have you publish your Postman API collection to the [Postman API Network](https://explore.postman.com/network/search?q=salesforce). It would put the collection within reach of the client of all our developers, as well as publicly on the network.

The option to publish to the network is available when publishing documentation from your collection in Postman -- let me know if you need any help!

[Great story BTW!!](https://developer.salesforce.com/blogs/2020/03/explore-the-salesforce-apis-with-a-postman-collection.html)

**Comment by pozil** [comment]
Hi @kinlane, thanks for reaching out and for the positive feedback.

@scolladon and I are both Salesforce employees and this contribution is still unofficial. We'd love to make it more visible but we need to make sure that we get approval before doing that.
We'll reach out to our management and product team and we'll keep you posted in this thread.

Thanks again.

**Comment by kinlane** [comment]
Understood. These types of things aren't easy to move forward. I figured I'd help get the juices flowing as I am being asked by your bosses to stimulate Salesforce and Postman working together. Let me know how I can help. I've planted my seeds with Salesforce leadership as well. ;-)

**Comment by pozil** [comment]
Just a quick FYI: I've opened the discussion with our product team. They/we are quite busy with the preparation of one of our major events (TrailheaDX) happening on June 25. Things won't move for the next couple of weeks but expect some news after that.

**Comment by kinlane** [comment]
Appreciate the update. Understood. Events suck the oxygen out of room.  I have also planted the seeds with a few folks I am working with to stimulate Salesforce + Postman partnership, so maybe it will trickle down. Appreciate the effort. Let me know if I can help here or in any other way.

**Comment by pozil** [comment]
Hi @kinlane, thanks for your patience. We discussed this opportunity with our API product teams and got positive feedback.

I've been reading your documentation for [adding an API to the API Network](https://learning.postman.com/docs/postman-for-publishers/api-network/add-api-network/) and I have a couple of questions regarding the process.

@scolladon and I would like to keep the collection as open source as possible while limiting the impact on the product teams.
Is this possible with the API Network? In other words, the API Network is a great tool to publish content but is there a way users could contribute to that content on top of consuming it?

**Comment by kinlane** [comment]
Good to hear @pozil 

There is the notion of forking a collection and then letting team members submit a pull request -- suggesting changes to a collection. However this feature is only available within workspaces and usually applied by internal teams. There isn't a model for managing this publicly with the community, but it is something that would be very interesting to brainstorm on for the road map -- accepting community contributions via collection.

Now, with that said. If the Postman collection is generated from an OpenAPI 3.0 specification, this artifact can be synced to a specific GitHub repository, allowing the public to submit changes via pull request to that repo -- any merged changes to the Open would then be synced back to the Postman workspace, and able to be synced with or regenerate your Postman collection, and any resulting documentation, mocks, or tests. 

Hope that makes sense! Happy to expand upon.

**Comment by kinlane** [comment]
Hey now. Just to bump this up for you guys. We'd love to have your collection in the network and have a big public API campaign in October we'd love to have you guys part of. Feel free to ping me directly at kin.lane@postman.com. I also wrote up a blog post on managing your OpenAPI publicly on GitHub -- I have been getting this question from a lot of API providers, so I wrote up as a post. I thought it might be relevant to y'alls approach as well.
